pkg/network/tcp/handshake.go

```diff
@@ -0,0 +1,56 @@
+package tcp
+
+import (
+	"boscoin.io/sebak/pkg/wire/message"
+	"net"
+	"time"
+)
+
+const (
+	connectTimeout = time.Second * 5
+)
+
+func (o *NetworkManager) handshake(conn net.Conn, inbound bool) (*peer, error) {
+	var err error
+	var their *message.HelloMessage
+	our := &message.HelloMessage{
+		ProtocolVersion: message.ProtocolVersion,
+		PubKey:          o.params.PubKey,
+		Timestamp:       uint32(time.Now().Second()),
+		Network:         o.params.NetworkPassphrase,
+	}
+
+	if inbound {
```

Comment:
What is the meaning of `inbound`?

Reply:
`inbound` is a incoming connection by accept loop.